### PR TITLE
 Qt 5.11: Append rocko as a supported layer

### PR DESCRIPTION
**Pull request by thenor**

### CODE REVIEW

**Comment by thenor** [comment]
Stopped the build since there are many builds in the queue and I can build manually first to test this out.

**Comment by Gagi2k** [comment]
not sure why, but that didn't work for me, i needed to remove the LAYERSERIES_COMPAT_qt5-layer in meta-qt5/conf/layer.conf to make it work

**Comment by thenor** [comment]
Yeah doesn't seem to have worked for me either.

**Comment by thenor** [comment]
This worked locally at least.

File: conf/variant/common/local.conf [hunk 1]
```diff
@@ -25,7 +25,8 @@ POKYQEMUDEPS_remove = "packagegroup-core-device-devel"
 NOHDD = "1"
 NOISO = "1"
 
-# Removing tar.bz2 image format as it is not used. Incase of need, it can be supplied through own local.conf
+# Removing tar.bz2 image format as it is not used.
+# If of needed it can be supplied through own local.conf
```
Comment (erikboto) [inline conf/variant/common/local.conf]: Grammar got weird, maybe remove the "of".

File: conf/variant/common/bblayers.conf [hunk 2]
```diff
@@ -1,3 +1,6 @@
+LAYERSERIES_CORENAMES_append = " rocko"
```
Comment (erikboto) [inline conf/variant/common/bblayers.conf]: Maybe a short comment on why this is done, since I guess it's quite unusual to change another layers config.

Comment (erikboto) [reply conf/variant/common/bblayers.conf]: (The comment above is meant for both lines here, not just the CORENAMES thing)

Comment (thenor) [reply conf/variant/common/bblayers.conf]: Oops I forgot to add my comment. I had it in another file, then moved it to this file.

File: conf/variant/common/local.conf (hunk 1 above)
Comment (thenor) [reply conf/variant/common/local.conf]: Yes. I will remove the "of".

**Pull request merged**